setup.py

```diff
@@ -1,38 +1,67 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
-"""
-Setup script for spyderplugins.p_autopep8
-"""
-from setuptools import setup, find_packages
+# -----------------------------------------------------------------------------
+# Copyright (c) Spyder Project Contributors
+#
+# Licensed under the terms of the MIT License
+# (see LICENSE.txt for details)
+# -----------------------------------------------------------------------------
 
+"""Setup script for spyder_vim"""
 
-def readme():
-    return str(open('README.rst').read())
+# Standard library imports
+import ast
+import os
 
+# Third party imports
+from setuptools import find_packages, setup
+
+HERE = os.path.abspath(os.path.dirname(__file__))
+
+
+def get_version(module='spyder_vim'):
+    """Get version."""
+    with open(os.path.join(HERE, module, '__init__.py'), 'r') as f:
+        data = f.read()
+    lines = data.split('\n')
+    for line in lines:
+        if line.startswith('VERSION_INFO'):
+            version_tuple = ast.literal_eval(line.split('=')[-1].strip())
+            version = '.'.join(map(str, version_tuple))
+            break
+    return version
+
+
+def get_description():
+    """Get long description."""
+    with open(os.path.join(HERE, 'README.rst'), 'r') as f:
+        data = f.read()
+    return data
+
+
+REQUIREMENTS = ['spyder>=3.2.0']
 
 setup(
-    name='spyder.vim',
-    version="0.1.0",
+    name='spyder_vim',
```

Comment:
It seems that the package is right: https://pypi.python.org/pypi/spyder-terminal